.eslintrc.js

```diff
@@ -47,41 +47,85 @@ module.exports = {
       files: ["*.{spec,test}.{js,jsx,ts,tsx}"],
```

Comment:
não é bloqueante e me desculpa por lembrar disso so agora <o>
maaaaas acho que rola sobreescrever as configs do eslint pela hierarquia das pastas.
poderiamos ter tipo um global na raiz (com as regras dos sonhos) e depois ter outros .eslintrc.js dentro dos pacotes flexibilizando o que for necessario (web e styles)